dbld/images/helpers/pip_packages.manifest

```diff
@@ -6,7 +6,6 @@ ply                     [centos, debian, ubuntu]
 pylint                  [centos-7, debian, ubuntu]
 
 # pip packages for python functional tests
-colorlog                [centos, debian, ubuntu]
```

Comment:
This first patch does not stand on its own. It breaks bisectibility. Can you move to the last commit?

Reply:
+1